src/agent0/ethpy/hyperdrive/interface/read_write_interface.py

```diff
@@ -194,7 +194,7 @@ async def async_open_long(
         """
         return await _async_open_long(
             interface=self,
-            agent=agent,
```

Comment:
I'm thinking we should rename the arg here (`read_write_interface.async_open_long` & elsewhere) to `sender` as well.

Reply:
The `_*` fns are meant to be at parity with the interface.